bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/BusEvent.java

```diff
@@ -77,14 +79,25 @@ static public Object sendCommand(String itemName, String commandString) {
             try {
                 Item item = registry.getItem(itemName);
                 Command command = TypeParser.parseCommand(item.getAcceptedCommandTypes(), commandString);
-                publisher.post(ItemEventFactory.createCommandEvent(itemName, command));
+                if (command != null) {
+                    publisher.post(ItemEventFactory.createCommandEvent(itemName, command));
+                } else {
+                    LoggerFactory.getLogger(BusEvent.class).warn(
```

Comment:
Since the logger is now used at least 2 times (also in line 91), maybe its time for storing it in a local variable than fetching it twice.

Reply:
hmmm, these are both exceptional cases which will occur rarely to never. I'm not really convinced that this justifies a static field. Remember, OSGi is a _dynamic_ module system.